diff --git a/CMakeLists.txt b/CMakeLists.txt
index 081ca6c097..a42718b9cd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,12 @@
 cmake_minimum_required(VERSION 3.12...3.16 FATAL_ERROR)
 
+if(VTK_USE_CCACHE)
+  find_program(CCACHE_PROGRAM ccache)
+  if(CCACHE_PROGRAM)
+    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
+  endif()
+endif()
+
 project(VTK)
 
 set(vtk_cmake_dir "${VTK_SOURCE_DIR}/CMake")
