diff --git a/CMakeLists.txt b/CMakeLists.txt
index e88d85a..6ba520d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,14 @@
 cmake_minimum_required(VERSION 3.8)
+
+# CCache block
+if(sqlite3_USE_CCACHE)
+  find_program(CCACHE_PROGRAM ccache)
+  if(CCACHE_PROGRAM)
+    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
+  endif()
+endif()
+
+# Project
 project(sqlite3
     VERSION 3
     LANGUAGES C
