--- numpy-1.16.2/numpy/distutils/fcompiler/__init__.py	2019-02-26 11:42:42.000000000 -0400
+++ NUMPY/numpy/distutils/fcompiler/__init__.py	2019-03-04 09:10:00.000000000 -0400
@@ -428,8 +428,8 @@ class FCompiler(CCompiler):
     def get_version(self, force=False, ok_status=[0]):
         assert self._is_customised
         version = CCompiler.get_version(self, force=force, ok_status=ok_status)
-        if version is None:
-            raise CompilerNotFound()
+        # if version is None:
+        #     raise CompilerNotFound()
         return version
